README.md

```diff
@@ -75,6 +75,10 @@ __What service nodes are failing?__
 
     sum by (node, service)(consul_catalog_service_node_healthy == 0)
 
+__What service checks are critical?__
+	
+	consul_health_service_status{status="critical"} == 1
```

Comment:
Nitpick: Let's use whitespace consistently, it seems the other code examples use 4 spaces instead one tab.